diagnostic.md

```diff
@@ -5,20 +5,28 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+      
+    (https://cloud.githubusercontent.com/assets/388761/12339386/dc1cc062-bae2-11e5-85be-ae33da715b2c.png)
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    component.js - this file is responsible for determining any attributes or
+    actions of the component.
+
+    template.js - this is where you will have an hbs call that displays
+    your component.
+
+    router.js - this is where you will specify the route to take in order to
+    display your component
```

Comment:
Yes.  The `template.hbs` is also responsible for binding data, and binding and dispatching actions.